src/main/fc/msp_fc.c

```diff
@@ -1015,12 +1015,21 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, sbuf_t *src, msp
         sbufWriteU8(dst, currentProfile->rcControlsConfig.alt_hold_deadband);
         sbufWriteU16(dst, masterConfig.flight3DConfig.deadband3d_throttle);
         break;
+
     case MSP_SENSOR_ALIGNMENT:
         sbufWriteU8(dst, masterConfig.sensorAlignmentConfig.gyro_align);
         sbufWriteU8(dst, masterConfig.sensorAlignmentConfig.acc_align);
         sbufWriteU8(dst, masterConfig.sensorAlignmentConfig.mag_align);
         break;
 
+    case MSP_ADVANCED_CONFIG:
+        sbufWriteU8(dst, masterConfig.gyroSync);
```

Comment:
Actually, Betaflight doesn't have gyroSync setting there which I would like to have. Ideas? Move it to the end of the message?